docs/integrations/xgboost.rst

```diff
@@ -51,100 +75,64 @@ Here is how to use the Neptune-XGBoost integration:
     xgb.train(params, dtrain, num_round, watchlist,
               callbacks=[neptune_callback()])  # neptune_callback is here
 
-|Example results|
-
 Logged metrics
-^^^^^^^^^^^^^^
+""""""""""""""
 These are logged for train and eval (or whatever you defined in the watchlist) after each boosting iteration.
 
 .. image:: ../_static/images/integrations/xgboost_metrics.png
    :target: ../_static/images/integrations/xgboost_metrics.png
    :alt: XGBoost overview
 
 Logged model
-^^^^^^^^^^^^
+""""""""""""
 The model (Booster) is logged to Neptune after the last boosting iteration. If you run cross-validation, you get a model for each fold.
 
 .. image:: ../_static/images/integrations/xgboost_model.png
    :target: ../_static/images/integrations/xgboost_model.png
    :alt: XGBoost overview
 
 Logged feature importance
-^^^^^^^^^^^^^^^^^^^^^^^^^
+"""""""""""""""""""""""""
 This is a very useful chart, as it shows feature importance. It is logged to Neptune as an image after the last boosting iteration. If you run cross-validation, you get a feature importance chart for each fold's model.
 
 .. image:: ../_static/images/integrations/xgboost_importance.png
    :target: ../_static/images/integrations/xgboost_importance.png
    :alt: XGBoost overview
 
-Logged visualized trees
-^^^^^^^^^^^^^^^^^^^^^^^
+Logged visualized trees (requires graphviz)
+"""""""""""""""""""""""""""""""""""""""""""
+Log first 6 trees at the end of training (tree with indices 0, 1, 2, 3, 4, 5)
+
+.. code-block:: python3
+
+    xgb.train(params, dtrain, num_round, watchlist,
+              callbacks=[neptune_callback(log_tree=[0,1,2,3,4,5])])
+
 Selected trees are logged to Neptune as an image after the last boosting iteration. If you run cross-validation, you get a tree visualization for each fold's model, independently.
 
+.. note::
```

Comment:
I'd put it up right at the top. 
I think we want to avoid the situation when somebody runs the code above and fails unexpectedly.

Reply:
agree. corrected